optimum_benchmark/backends/pytorch.py

```diff
@@ -38,32 +46,25 @@
 @dataclass
 class PyTorchConfig(BackendConfig):
     name: str = "pytorch"
-    version: str = torch.__version__
+    version: str = torch_version
```

Comment:
What is the point?

Reply:
now it's handled by resolvers and `importlib` to avoid importing torch.